indico/modules/events/editing/controllers/base.py

```diff
@@ -64,3 +67,13 @@ def _process_args(self):
                          .with_parent(self.contrib)
                          .filter_by(type=self.editable_type)
                          .first())
+
+
+class RHEditablesBase(RHEditingManagementBase):
+    """Base class for operations on multiple editables."""
+
+    def _process_args(self):
+        RHEditingManagementBase._process_args(self)
+        self.editables = parser.parse({
+            'editables': EditableList(required=True, validate=not_empty)
```

Comment:
should work with the decorator as well

Reply:
Ah no, you need to ensure that you cannot pass editable ids which do not belong to the given event!

So, the decorator may not work after all unless you can do it with the context coming from the RH...